README.md

```diff
@@ -1,10 +1,33 @@
+# NimbleTemplate
+
+[![Test template](https://github.com/nimblehq/elixir-templates/actions/workflows/test_template.yml/badge.svg?branch=develop)](https://github.com/nimblehq/elixir-templates/actions/workflows/test_template.yml)
+
+
+> A Library for initialize new Phoenix projects
+
 ## Introduction
 
-Phoenix/Mix template for projects at [Nimble](https://nimblehq.co/).
+NimbleTemplate is a library for initializing new Phoenix projects at [Nimble](https://nimblehq.co/). 
+It saves us a lot of time on the installation and configuration of new Phoenix projects.
```

Comment:
I'm thinking we could remove this phase or we need to rephase it a bit something like `Speed up the initialize project process`...